src/include/ipc/topology.h

```diff
@@ -46,6 +46,7 @@ enum sof_comp_type {
 	SOF_COMP_DEMUX,
 	SOF_COMP_ASRC,		/**< Asynchronous sample rate converter */
 	SOF_COMP_DCBLOCK,
+	SOF_COMP_CROSSOVER,
```

Comment:
@plbossart @kv2019i @mmaka1 is the driver still accepting new types or should all new stuff be using a generic "PROCESS" type now ? If so what's the ETA on this new process type landing in the kernel ?

Reply:
@lgirdwood can we land this then fix this up later? There are more PRs coming in using the old comp_type (see #2925) and this is functionally correct with the current state of the system. I will own updating this if need be to the new process type. As mentioned above, any changes now would require this PR to be closed and a new one to be opened therefore losing all history of comments.

Reply:
@plbossart are you still accepting new types currently (or is the process type ready for merging) ? we have 3 PRs blocking on new types atm iirc